### ISSUE TITLE
[2.1] 6.6.2 inconsistent writing of where rules are found

### ISSUE THREAD

**Issue by StephenOTT** [oasis-tcs/cti-stix2#179]
In 6.6.2 the rule of "body or body_raw_ref must be included" is found at the top of the section.  In order sections rules such as this are found in the specific property descriptions.  Suggest we standardize here unless there is a reason for not having it in the same place?

**Comment by rpiazza** [comment]
Other sections (4.11, 4.13, 6.1, 6.7, 6.12) have normative statements about multiple property constraints in the text, but only 6.1 and 4.13 also has something in the property table.  I wonder if that was because in 6.1 and 4.13, the constraint is for one or the other, but not both.  @jordan2175 - do you remember if we did this on purpose?

**Comment by jordan2175** [comment]
In general we tried to put them in the top description, when they were blanket statements like this.  I am sure there are a few places where we are not consistent, but that is what we tried to do.  In some specific cases I know we choose to put them in the table as we felt they needed to be there.  So the more specific the statement the more likely it is in the property table.  If the statement is more general about the object as a whole, then it is located in the description.
